spec/controllers/book_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
```

Comment:
Sao không lấy cái controller đủ action mà làm em ơi

Reply:
@duypn-2245 dạ controller này chỉ có 1 action nên nó hơi it. pull controller khác em đẩy pull nhiều action ạ